src/components/varsler/Details.tsx

```diff
@@ -0,0 +1,145 @@
+import { ErrorMessage, HStack, Heading, Skeleton, VStack } from '@navikt/ds-react';
+import { getRouteApi } from '@tanstack/react-router';
+import { Suspense } from 'react';
+import Card from 'src/components/Card';
+import ErrorBoundary from 'src/components/ErrorBoundary';
+import { useFilterVarsler } from 'src/components/varsler/List/utils';
+import type { FeiletVarsling, Varsel } from 'src/lib/types/modiapersonoversikt-api';
+import { ENDASH, emptyReplacement, formaterDato } from 'src/utils/string-utils';
+import { twMerge } from 'tailwind-merge';
+
+const routeApi = getRouteApi('/new/person/varsler');
+
+const FeilteVarslingerListe = ({
+    tittel,
+    feilteVarslinger
+}: {
+    tittel: string;
+    feilteVarslinger: FeiletVarsling[];
+}) => {
+    return (
+        <div className="my-1">
+            <div className="font-bold">{tittel}</div>
```

Comment:
```suggestion
            <BodyShort fontWeight="bold">{tittel}</BodyShort>
```

e.g. (not the correct prop, just an example)